src/components/molecules/ModalConfirmWithNamespaceSelect/ModalConfirmWithNamespaceSelect.tsx

```diff
@@ -115,6 +116,10 @@ const ModalConfirmWithNamespaceSelect: React.FC<IProps> = props => {
     }
   }, [defaultOption, defaultNamespace, namespaces]);
 
+  useEffect(() => {
+    setDisableNamespaces(!resources.some(r => r.kind !== 'Namespace'));
```

Comment:
Here it would be better to create a useMemo instead of state + useEffect:

```ts
const areNamespacesDisabled = useMemo(() => resources.some(r => r.kind === 'Namespace'), [resources]);
```

Reply:
thanks @devcatalin - updated and added some additional logic to provide a (marginally) better UX